<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="mainupLayout">
        <item>
         <layout class="QHBoxLayout" name="titleLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>168</width>
              <height>38</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="funcLayout">
          <item>
           <widget class="QPushButton" name="minButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="maxButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="closebutton">
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>16</height>
             </size>
            </property>
            <property name="autoRepeatDelay">
             <number>297</number>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>200</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="loginLayout">
             <property name="spacing">
              <number>40</number>
             </property>
             <item>
              <widget class="QLineEdit" name="actEdit">
               <property name="placeholderText">
                <string>输入你的账号</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="pwdEdit">
               <property name="placeholderText">
                <string>输入你的密码</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="BtnLayout">
               <property name="spacing">
                <number>10</number>
               </property>
               <item>
                <widget class="QPushButton" name="loginBtn">
                 <property name="text">
                  <string>登录</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="registerBtn">
                 <property name="text">
                  <string>注册</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>200</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
